src/containers/AccountConnection.jsx

```diff
@@ -120,15 +121,15 @@ class AccountConnection extends Component {
       })
   }
 
-  updateAccount (connector, account, values) {
+  updateAccount (connector, account, values, successTimeout) {
     Object.assign(account.auth, values)
 
     this.setState({ submitting: true })
 
     return this.store.updateAccount(connector, account, values)
     .then(account => {
       this.setState({ account: account })
-      return this.store.runAccount(connector, account)
+      return this.store.runAccount(connector, account, successTimeout)
```

Comment:
Ah it is set in https://github.com/cozy/cozy-collect/pull/156/files#diff-6b594c34e36119c213361a6409f630b8R16 and https://github.com/cozy/cozy-collect/pull/156/files#diff-6b594c34e36119c213361a6409f630b8R37. Or I do not get your point.